floater/utils.py

```diff
@@ -251,7 +252,7 @@ def floats_to_netcdf(input_dir, output_fname,
 
     match_pattern = float_file_prefix + '.*.csv'
     float_files = glob(os.path.join(input_dir, match_pattern))
-    float_header = dd.read_csv(float_files[0]).columns
+    float_header = pd.read_csv(float_files[0], nrows=0).columns
```

Comment:
Yes. I tested it using different values 2, 1, and 0, and all of them work. So I picked 0 here.